programs/github

```diff
@@ -1,15 +1,28 @@
 local github = dofile("apis/github")
+local passed = {...}
```

Comment:
yeah derp... had to move it to top of file to avoid stupid vararg function error.